CMakeLists.txt

```diff
@@ -27,6 +27,13 @@
 cmake_minimum_required(VERSION 2.8)
 project(Unvanquished C CXX)
 
+# When we build in the source dir, the daemon executable cannot be create
+# as there is already a daemon directory, so we disallow building in the
+# source dir.
+if(CMAKE_SOURCE_DIR STREQUAL CMAKE_BINARY_DIR)
```

Comment:
Do you think it's worth having a smarter comparison which accounts for symlinks and whatnot?